bot/connector-web/src/test/resources/card_with_buttons.json

```diff
@@ -20,6 +20,7 @@
             "title": "title",
             "url": "http://www.sncf.com",
             "type": "web_url",
+            "openInSameWindow": "false",
```

Comment:
`"openInSameWindow": false,` sans les quotes ? (pour être iso les autres fichiers de test)

Reply:
@francoisno please check the new commit

Reply:
I just pushed a fix of test resources (2 `openInSameWindow` remained).
Please let me know if it's ok for you @bakic